Makefile

```diff
@@ -1,36 +1,43 @@
 NAMESPACE = connaisseur
-IMAGE_REPOSITORY := $(shell yq e '.kubernetes.deployment.image.repository' helm/values.yaml)
-VERSION := $(shell yq e '.appVersion' helm/Chart.yaml)
-COSIGN_VERSION = 2.2.2
+IMAGE_REPO := $(shell yq e '.kubernetes.deployment.image.repository' charts/connaisseur/values.yaml)
+VERSION := $(shell yq e '.appVersion' charts/connaisseur/Chart.yaml)
+KD := kubernetes.deployment
+ALL_RS := all,mutatingwebhookconfigurations,clusterroles,clusterrolebindings,configmaps,secrets,serviceaccounts,crds
 
-.PHONY: all docker install uninstall upgrade annihilate
-
-all: docker install
+.PHONY: docker install uninstall annihilate test install-dev upgrade lint
 
+# meant for local building of docker image
 docker:
-	docker build --pull --build-arg COSIGN_VERSION=$(COSIGN_VERSION) -f docker/Dockerfile -t $(IMAGE_REPOSITORY):v$(VERSION) .
+	docker buildx build --pull -f build/Dockerfile -t $(IMAGE_REPO):v$(VERSION) .
 
 install:
-	#
-	#=============================================
-	#
-	# The installation may last up to 5 minutes.
-	#
-	#=============================================
-	#
-	helm install connaisseur helm --atomic --create-namespace --namespace $(NAMESPACE)
-
-dev-install:
-	helm install --set kubernetes.deployment.replicasCount=1,kubernetes.deployment.imagePullPolicy=Never,application.logLevel=DEBUG connaisseur helm --atomic --create-namespace --namespace $(NAMESPACE)
+	helm install connaisseur charts/connaisseur --atomic --create-namespace --namespace $(NAMESPACE) $(HELM_ARGS)
+
+install-dev:
+	helm install --set $(KD).replicasCount=1,$(KD).imagePullPolicy=Never,application.logLevel=debug \
+	 connaisseur charts/connaisseur --atomic --create-namespace --namespace $(NAMESPACE) $(HELM_ARGS)
 
 uninstall:
 	helm uninstall connaisseur -n $(NAMESPACE)
 	kubectl delete ns $(NAMESPACE)
 
-upgrade:
-	helm upgrade connaisseur helm -n $(NAMESPACE) --wait
-
 annihilate:
-	kubectl delete all,mutatingwebhookconfigurations,clusterroles,clusterrolebindings,configmaps,secrets,serviceaccounts,crds -lapp.kubernetes.io/instance=connaisseur -n $(NAMESPACE)
+	kubectl delete $(ALL_RS) -lapp.kubernetes.io/instance=connaisseur -n $(NAMESPACE)
 	kubectl delete imagepolicies -lapp.kubernetes.io/instance=connaisseur -n $(NAMESPACE) || true
 	kubectl delete ns $(NAMESPACE)
+
+test:
+	go test ./... -race -coverprofile=cover.out -covermode=atomic; go tool cover -func cover.out
+
+upgrade:
+	helm upgrade connaisseur charts/connaisseur --namespace $(NAMESPACE) --wait
+
+kind-int-test:
+	./tests/integration/run_integration_tests.sh -c kind -r "regular cosign"
+
+# TODO: probably remove before release
```

Comment:
We should either rm the command or the TODO. I'm kinda happy with the cmd tbh, so would be fine keeping it